lbrynet/core/LBRYWallet.py

```diff
@@ -368,6 +368,19 @@ def _get_id_for_return(claim_id):
         d.addCallback(_get_id_for_return)
         return d
 
+    def get_my_claim(self, name):
+        def _get_claim_for_return(claim):
+            if not claim:
+                return False
+            d = self.get_claim(name, claim['claim_id'])
+            d.addCallback(lambda c: self._format_claim_for_return(name, c, claim['txid']))
+            return d
+
+        d = self.get_name_claims()
+        d.addCallback(lambda claims: next((c for c in claims if c['name'] == name and not c['is spent']), False))
```

Comment:
this is doing too much on one line, I'd make a separate function: `d.addCallback(find_first_unspent, name)`
and, of course, unit test it.

Should it be `'is_spent'` instead of `'is spent'`